spec/mailers/user_mailer_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, type: :mailer do
+  before(:each) do
+    FactoryGirl.create :user
+  end
+
+  it "should deliver mail containing password" do
+    expect{UserMailer.send_password(User.last).deliver_now}.to change {ActionMailer::Base.deliveries.count}.by(1)
```

Comment:
Thanx. Can you please have a look at this? [stackoverflow.com/questions/25227196/expected-count-to-have-changed-by-1-but-was-not-given-a-block](url) . Even when I use curve brackets in the expect part it generates an error.